openwisp_ipam/tests/test_multitenant.py

```diff
@@ -48,3 +53,381 @@ def test_multitenancy_subnet_queryset(self):
             visible=[data['subnet1']],
             hidden=[data['subnet2']],
         )
+
+
+class TestMultitenantApi(
+    TestMultitenantAdminMixin, CreateModelsMixin, PostDataMixin, TestCase
+):
+    def setUp(self):
+        super().setUp()
+        # Creates a manager for each of org_a and org_b
+        org_a = self._create_org(name='org_a', slug='org_a')
+        org_b = self._create_org(name='org_b', slug='org_b')
+        user_a = self._create_operator(
+            username='user_a',
+            email='usera@tester.com',
+            password='tester',
+            is_staff=True,
+        )
+        ou = OrganizationUser.objects.create(user=user_a, organization=org_a)
+        ou.is_admin = True
+        ou.save()
+        user_b = self._create_operator(
+            username='user_b',
+            email='userb@tester.com',
+            password='tester',
+            is_staff=True,
+        )
+        ou = OrganizationUser.objects.create(user=user_b, organization=org_b)
+        ou.is_admin = True
+        ou.save()
+        # Creates a superuser
+        self._create_operator(
+            username='superuser',
+            email='superuser@tester.com',
+            password='tester',
+            is_superuser=True,
+        )
+
+    def test_subnet(self):
+        org_a = self._get_org(org_name='org_a')
+        subnet = self._create_subnet(subnet='10.0.0.0/24', organization=org_a)
+
+        with self.subTest('Test subnet accessible by org manager'):
+            self._login(username='user_a', password='tester')
+            response = self.client.get(reverse('ipam:subnet', args=(subnet.id,)))
```

Comment:
let's avoid duplicating `reverse('ipam:subnet', args=(subnet.id,)` in every subtest.
It's a small improvement, but nonetheless I want you to get used to the idea of avoiding code duplication, also during tests, even though in tests a bit of duplication is widely accepted, but still is good to avoid it when it's cheap like in this case.

This applies to all other occurrences of the same issue in these tests, if a value is reused in different subtests or in different tests, try to store it in a variable which is reused.

Reply:
Done :+1: